frontend/views/containers/proposals/Mincome.vue

```diff
@@ -1,20 +1,121 @@
-<template lang="pug">
-proposal-template(
-  :subtitle="L('Change minimum income')"
-  :onsubmit='handleSubmit'
-)
-  | I step to change minimum income it&apos;s on the way.
+<template lang='pug'>
+  proposal-template(
+    title= 'L("Change minimum income")'
+    footer='L("According to your voting rules, 8 out of 10 members will have to agree with this.")"'
+    :disabled='$v.form.$invalid || ($v.steps[config.steps[currentStep]] && $v.steps[config.steps[currentStep]].$invalid)'
+    :maxSteps='config.steps.length'
+    :currentStep.sync='currentStep'
+    @submit='L("submit")'
+  )
+
+    .field(v-if='currentStep === 0' key='0')
+      label.label
+        i18n New minimum income
+      .select-wrapper
+        input.input(
+          ref='mincome'
+          type='number'
+          placeholder='Amount'
+          name='incomeProvided'
+          step='1'
+          min='0'
+          required=''
+          :class="{ 'error': $v.form.incomeProvided.$error }"
+          v-model='form.incomeProvided'
+          @keyup.enter='next'
+        )
+
+        select(
+          name='incomeCurrency'
+          required=''
+          v-model='form.incomeCurrency'
+        )
+          option(
+            v-for='(symbol, code) in currencies'
+            :value='code'
+            :key='code'
+          ) {{ symbol }}
+
+      p
+        i18n Currently $1000 monthly.
+
+    .field(v-if='currentStep === 1' key='1')
+      label.label
+        i18n Why are you proposing this change?
+
+      textarea.textarea(
+        name='changeReason'
+        ref='purpose'
+        :placeholder='L("The reason why I\' propositiong this change is...")'
```

Comment:
Now that the text is double-quoted, do you still need to escape the apostrophe here?

Reply:
yes